doc-content/enterprise-only/admin-and-config/business-central-settings-changing-permissions-proc.adoc

```diff
@@ -4,13 +4,21 @@
 You cannot change permissions for an individual user. However, you can change permissions for groups and roles. The changed permissions apply to users that have the role or belong to the group that you changed.
 
 .Procedure
-. Click the gear icon, and then click *Roles* or *Groups*.
+. In {CENTRAL} and click on *Admin* -> *Roles* or *Groups*.
```

Comment:
Missing something. In Central do what and click? Click, never click on.

Reply:
Changed it to -> Log in to {CENTRAL} and click ...... (removed "on").